src/test/java/seedu/address/model/tag/MemberTest.java

```diff
@@ -16,16 +16,31 @@ public void constructor_null_throwsNullPointerException() {
 
     @Test
     public void constructor_invalidMemberName_throwsIllegalArgumentException() {
-        String invalidMemberName = "";
+        final String invalidMemberName = "";
         assertThrows(IllegalArgumentException.class, () -> new Member(invalidMemberName));
+
+        final String invalidMemberNameTwo = "    ";
+        assertThrows(IllegalArgumentException.class, () -> new Member(invalidMemberNameTwo));
+
+        final String invalidMemberNameThree = "mary/";
+        assertThrows(IllegalArgumentException.class, () -> new Member(invalidMemberNameThree));
```

Comment:
Should this be put into a different test method? This method tests if the member name is blank, but the highlighted portion is responsible for testing if `/` is permitted, which is in a different equivalence partition.

It might be more beneficial to move this to a different method to make the tests more clear!

Reply:
Oh hmmm thanks for the feedback. I was thinking of this test testing both scenarios in which the Strings are invalid member names